ironfish/src/blockchain/blockchain.ts

```diff
@@ -1015,39 +985,23 @@ export class Blockchain<
       [this.nullifiers.counter, this.nullifiers.leaves, this.nullifiers.nodes],
```

Comment:
Same comment needs to be updated here (should have been `looseNullifiers` anyway 😅 )